src/ecdf.jl

```diff
@@ -0,0 +1,21 @@
+
+# ---------------------------------------------------------------------------
+# empirical CDF
+
+@recipe function f(ecdf::StatsBase.ECDF)
+    seriestype --> :path
+    linetype := :steppost
```

Comment:
These two seem to contradict each other - does `linetype := :steppost` have an effect?

Reply:
Oh, I hadn't seen that `linetype` was an alias for `seriestype`. I removed the first definition, as `linetype := :steppost` is required for the plot to jump in the right places.